ciere/json/detail/value_impl.hpp

```diff
@@ -45,6 +45,7 @@ namespace ciere { namespace json
          static R apply( T const & v
                        , typename boost::enable_if<boost::is_convertible<R,T> >::type* dummy=0 )
          {
+            (void)dummy;
             return v;
          }
 
```

Comment:
Wat? It will be a compile error in that case and it will be immediately obvious.